packages/logs/lib/client.integration.test.ts

```diff
@@ -5,11 +5,11 @@ import { beforeAll, describe, expect, it, vi } from 'vitest';
 import { deleteIndex, migrateMapping } from './es/helpers.js';
 import { indexMessages } from './es/schema.js';
 import { logContextGetter } from './models/logContextGetter.js';
-import * as model from './models/messages.js';
-import { getOperation, listMessages, listOperations } from './models/messages.js';
+import * as modelMessages from './models/messages.js';
+import * as modelOperations from './models/operations.js';
```

Comment:
would it be possible to name the imports `Operations` and `Messages` with capital letter?